### PR TITLE
Add deprecation warning

### PR DESCRIPTION
**Pull request by noccor**
Let people know Intercom is being deprecated

### CODE REVIEW

**Comment by noccor** [comment]
@roytomeij - how do we format dates here?

@jvanbaarsen - is there anything we should advise them to bridge the intercom gap, or is it not worth it being such a small group of users?

**Comment by jvanbaarsen** [comment]
@noccor I'd say it's not worth it.

File: source/application/integrations/intercom.html.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ description: Intercom is one place for your entire business to see and talk to c
 layout: integration
 ---
 
+!> ⚠️ On October 1st 2022, we will discontinue all support for our Intercom integration. From this date onwards, Intercom integrations will no longer work with AppSignal. If your application currently uses the AppSignal Intercom Integration, we advise removing the configuration from your application's codebase before the feature is deprecated.
```
Comment (roytomeij) [inline source/application/integrations/intercom.html.md:7]: Once, it's "Intercom integration," and once, it's "Intercom Integration" (with a capital I).

**Review by roytomeij — CHANGES_REQUESTED** [review]

**Comment by roytomeij** [comment]
> @roytomeij - how do we format dates here?

"August 4, 2022": with a comma in between the day and the year.

File: source/application/integrations/intercom.html.md (hunk 1 above)
Comment (roytomeij) [inline source/application/integrations/intercom.html.md:7]:
```suggestion
!> ⚠️ On October 1st, 2022, we will discontinue all support for our Intercom integration. From this date onwards, Intercom integrations will no longer work with AppSignal. If your application currently uses the AppSignal Intercom Integration, we advise removing the configuration from your codebase before the feature is deprecated.
```

**Pull request merged**